### ISSUE TITLE
Implement per column formatting settings

### ISSUE THREAD

**Issue by philipbower122** [LiveSplit/LiveSplitOne#281]
I cannot find the option to set this section to change decimal places for this
![Layout pic](https://user-images.githubusercontent.com/38558297/75865614-b8fd3d80-5dfb-11ea-88cd-fc837c637cbf.PNG)

**Comment by CryZe** [comment]
Yeah this can't be configured yet.

**Comment by philipbower122** [comment]
Why not?

**Comment by CryZe** [comment]
Because no one implemented this yet. Not every setting has been ported over from the original LiveSplit yet.

**Comment by philipbower122** [comment]
ah so. to add it i just get the DLL file from the App and import it into the online version

**Comment by CryZe** [comment]
No, LiveSplit One shares basically no code (directly) with the original LiveSplit. DLLs are specific to windows (more or less) so you can't just load them in LiveSplit One. We need to properly implement this setting for LiveSplit One, there's nothing you can do atm.

**Comment by philipbower122** [comment]
Ok thanks for clarifying. I will close this ticket just keep updated for if it gets sorted.

**Comment by CryZe** [comment]
I'd want to keep this open so we don't forget about it.

**Comment by philipbower122** [comment]
ok
